addons/web_editor/views/snippets.xml

```diff
@@ -306,20 +306,6 @@
                     <we-button data-shape="web_editor/Zigs/04" data-select-label="Zigs 04"/>
                     <we-button data-shape="web_editor/Zigs/05" data-select-label="Zigs 05"/>
                 </we-select-page>
-                <we-select-page string="Bold">
```

Comment:
Convention for commit messages:
- the title should complete the sentence "When applied, this commit will ...", so "swap" instead of "swaps" in your title :+1: 
- For PR which contain multiples commits, you have to add a link of the PR in each commit (I personally write `Part of https://github.com/odoo/odoo/pull/59613` just above the `task-...`

Reply:
Oh... so I should actually create the PR before doing the commits. Or is there a way to update the commit titles afterward ?

Reply:
No, you must have commits to create the PR of course, but then you can amend them one by one to change the message to include the PR link. This is to be able to find the PR again once the commits are merged.

Normally when you work on a task:
- At the end the first day: create the PR with your work in *draft* state
- At the end of each day (at least): push your work on the PR
- When it is finished, clean your git history to have nice meaningful commits which mention the PR
- Then ask for review